client/src/main/kotlin/no/nav/su/se/bakover/client/oppdrag/utbetaling/UtbetalingRequestMapper.kt

```diff
@@ -57,6 +57,12 @@ internal fun toUtbetalingRequest(
                                 refDelytelseId = null,
                                 refFagsystemId = null,
                                 attestant = listOf(UtbetalingRequest.Oppdragslinje.Attestant(utbetaling.behandler.navIdent)),
+                                grad = it.uføregrad?.let { uføregrad ->
```

Comment:
Kan godt `!!` bang bange her. Den bør finnes i det vi gjør denne requesten.

Reply:
Det kan vel fort tenkes at denne ikke nødvendigvis eksisterer ved endring. Så vidt jeg kan se i `UtbetalingServiceImpl` gjør vi ikke noe for å legge ved uføregrad ved opprettelse av `Utbetalingslinje.Endring` (opphør, stans, reaktivering) - `!!` her vil å så måte ikke fungerer for eventuelle endringer av linjer som ble opprettet før denne PRen.

Reply:
@jacob-meidell Kan vel tenkes at vi kunne hentet det fra behandlingen. Vi har jo dataene.

Reply:
Det bør være mulig for oss å fiske det ut fra behandlingen og legge det til hvor vi evt måtte behøve det. Et spørsmål som bør besvares før vi evt gjør noe slikt er om OS kommer til å godta det. Endringene gjenspeiler endring av eksisterende linjer.. er usikker på hvordan OS kommer til å reagere dersom vi plutselig legger til `grad` på en linje som hos dem ikke har dette feltet (bør testes i simulering/SOAP først).

Reply:
vi har gått for å ikke legge på uføregrad på eksisterende utbetalinger :)